src/Controls/src/Core/TapGestureRecognizer.cs

```diff
@@ -42,17 +46,22 @@ public int NumberOfTapsRequired
 			set { SetValue(NumberOfTapsRequiredProperty, value); }
 		}
 
-		public event EventHandler Tapped;
+		public ButtonsMask Buttons
+		{
+			get { return (ButtonsMask)GetValue(ButtonsProperty); }
+			set { SetValue(ButtonsProperty, value); }
+		}
+
+		public event EventHandler<TappedEventArgs>? Tapped;
```

Comment:
So @PureWeen was this meant to be a breaking change hosing any .NET 6 components? 
<img width="449" alt="image" src="https://github.com/dotnet/maui/assets/1378165/1a4c1d16-f188-4c10-bf89-e43c898cd81b">